<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>1015</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eunomia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Booking-on point servers:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="bookingOnPointList" name="depotList">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current server status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ServerStatusTerminal" name="serverStatus">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(192, 191, 188);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>352</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="stopPollingButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop Polling</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startPollingButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start Polling</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionStart_Server"/>
    <addaction name="actionStop_Server"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Polling"/>
    <addaction name="actionStop_Polling"/>
    <addaction name="separator"/>
    <addaction name="actionStart_All_Polling"/>
    <addaction name="actionStop_All_Polling"/>
    <addaction name="actionHalt_All_Servers"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure_Server"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuMaintenance">
    <property name="title">
     <string>Maintenance</string>
    </property>
    <addaction name="actionRestart_Server"/>
    <addaction name="actionReload_Configuration"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Log"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMaintenance"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConfigure_Server">
   <property name="text">
    <string>Configure Server...</string>
   </property>
  </action>
  <action name="actionStart_Server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run Server</string>
   </property>
  </action>
  <action name="actionStop_Server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Halt Server</string>
   </property>
  </action>
  <action name="actionStart_Polling">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start Server Polling</string>
   </property>
  </action>
  <action name="actionStop_Polling">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Server Polling</string>
   </property>
  </action>
  <action name="actionRestart_Server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Full Restart</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionStart_All_Polling">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start All Polling</string>
   </property>
  </action>
  <action name="actionStop_All_Polling">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop All Polling</string>
   </property>
  </action>
  <action name="actionHalt_All_Servers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Halt All Servers</string>
   </property>
  </action>
  <action name="actionSave_Log">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Log...</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates...</string>
   </property>
  </action>
  <action name="actionReload_Configuration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload Configuration</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>bookingOnPointList</class>
   <extends>QListWidget</extends>
   <header>../../../bookingOnPointList.hpp</header>
  </customwidget>
  <customwidget>
   <class>ServerStatusTerminal</class>
   <extends>QListView</extends>
   <header>../../../server_status_terminal.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
